tests/Neo.SmartContract.Framework.UnitTests/TestClasses/Contract_Storage.cs

```diff
@@ -1,7 +1,9 @@
 using Neo.SmartContract.Framework.Services;
+using Neo.SmartContract.Framework;
 
 namespace Neo.SmartContract.Framework.UnitTests.TestClasses
 {
+
     public class Contract_Storage : SmartContract
```

Comment:
@Liaojinghui  you love this empty lines, isn't it?

```suggestion
    public class Contract_Storage : SmartContract
```